data/coalition weapons.txt

```diff
@@ -112,7 +112,7 @@ outfit "Finisher Pod"
 		"radar tracking" .8
 		"missile strength" 50
 		"submunition" "Active Finisher" 1
-	description `Finisher Torpedoes are one of the Heliarchs' most dreaded weapons. In fact, in one way or another most of their other weapons merely exist to hold a ship in place for long enough for a barrage of Finishers to destroy it.`
+	description `Finisher Torpedoes are one of the Heliarchs' most dreaded weapons. Made for destroying Quarg warships, Finishers deal an enormous amount of damage, though they are overkill against less powerful vessels. In one way or another, most of the Heliarchs' other weapons merely exist to hold a ship in place for long enough for a barrage of Finishers to destroy it.`
```

Comment:
In general, I try to avoid ship and outfit descriptions that do nothing but reiterate the information you can get from the weapon's stats. It's much better for the descriptions to provide some sort of flavor text (historical information, etc.) that is interesting or funny in its own right.